src/transforms/globalCssToCssModule/ts/getClassNameNodeReplacement.ts

```diff
@@ -0,0 +1,67 @@
+import { ts, NodeParentType, SyntaxKind } from 'ts-morph'
+import {
+    Expression,
+    StringLiteral,
+    CallExpression,
+    JsxExpression,
+    ComputedPropertyName,
+    PropertyAccessExpression,
+} from 'typescript'
+
+import { wrapIntoClassNamesUtility } from './classNamesUtility'
+
+export interface GetClassNameNodeReplacementOptions {
+    parentNode: NodeParentType<StringLiteral>
+    leftOverClassName: string
+    exportNameReferences: PropertyAccessExpression[]
+}
+
+function getClassNameNodeReplacementWithoutBraces(
+    options: GetClassNameNodeReplacementOptions
+): PropertyAccessExpression | CallExpression {
+    const { leftOverClassName, exportNameReferences } = options
+
+    // We need to use `classNames` utility for multiple `exportNames` or for a combination of the `exportName` and `StringLiteral`.
+    // className={classNames('d-flex mr-1 kek kek--primary')} -> className={classNames('d-flex mr-1', styles.kek, styles.kekPrimary)}
```

Comment:
@valerybugakov do we ignore all classes within the className prop if we don't have them in the CSS file?